docs/testbed/README.testbed.Example.Config.md

```diff
@@ -0,0 +1,143 @@
+# Example of Testbed Configuration
+Here is an example of how to manually get the testbed information when you want to allocate some tests on the DUTs. It includes how to get the testbed IP address, location, and how to find where is the PTF docker from the configuration files.
+
+## Testbed Inventory
+
+- [```ansible/lab```](/ansible/lab): Include all lab DUTs, fanout switches, and testbed server topologies
+- [```ansible/veos```](/ansible/veos): all servers and VMs
+- [```ansible/testbed.csv```](/ansible/testbed.csv) is the topology configuration file for the testbed (Will be deprecated).
+- [```ansible/testbed.yaml```](/ansible/testbed.yaml) is the topology configuration file for the testbed .
+- [```ansible/str```](/ansible/str) [```ansible/str2```](/ansible/str2)  is the environment configuration files for the testbed.
+
+
+## Testbed example
+Say here, we want to use the DUT as the name `str-s6000-acs-12`, the testbed with the name `vms13-4-t0`.
+Then we can get the related information from [```ansible/testbed.csv```](/ansible/testbed.csv) and [```ansible/testbed.yaml```](/ansible/testbed.yaml).
```

Comment:
You can use `ansible/vtestbed.csv` and `ansible/vtestbed.yaml` as examples. The inventory file for public VS setup testbeds is `ansible/veos.vtb`. This inventory file contains not just VM and server information, it also contains DUT information.